docs/extensions/logging-and-monitoring.md

```diff
@@ -90,31 +97,34 @@ data:
 
 ## Logging
 
-The central fluentd/fluent-bit instances in the `garden` namespace are parsing the logs from all containers in the seed cluster.
-The shoot-specific instances only extract exactly those logs for the pods of the control plane they are responsible for.
-This allows to only fetch the logs for the pods once for the whole cluster, and to distribute them afterwards.
+In Kubernetes clusters, container logs are non-persistent and do not survive stopped and destroyed containers. Gardener addresses this problem for the components that it manages, by introducing its own managed logging solution. It is integrated with the Gardener monitoring stack to have all troubleshooting context in one place.
 
-Extension controllers might deploy components as part of their reconciliation next to the shoot's control plane.
-Examples for this would be a cloud-controller-manager or CSI controller deployments.
-In some cases, the extensions want to submit logging parse configuration for these components such that their logs can be scraped by Gardener's fluentd/fluent-bit deployment(s), and later be visible in the Kibana dashboards.
+!["Cluster Logging Topology"](https://grafana.com/static/assets/img/blog/new-logging-architecture.png?src=tw&mdm=social "Cluster Logging Topology")
 
-:warning: As their is only the central fluentd/fluent-bit deployment (and not a shoot-specific deployment like in the case of monitoring, see above) the logging parse configuration must be only provided once and **not per shoot namespace**.
-Also, as fluentd/fluent-bit parses the logs based on the container name you should make sure that the container names inside your provider-specific pods are prefixed with your extension name.
+Gardener logging consists of:
+- Fluent-bit daemonsets on each seed node, collecting logs from their node. A custom plugin then takes care to distribute the log messages to the Loki instances deployed in the shoot that they are intended for. This allows to fetch the logs once for the whole cluster, and to distribute them afterwards.
+- One Loki statefulset in the `garden` namespace, which contains logs from the seed cluster and one per each shoot namespace, which contains logs from shoot's control plane.
```

Comment:
I would start with the Lokis in the shoot namespces, and then say that the Loki in the garden namespace store rest of the logs - e.g. the one from kube-system, garden, extension-* and other namespaces.

Reply:
Please, check again this one.

Reply:
Yep, still WiP and considering this suggestion plus a couple of more improvements before merging. Probably next week.